nevergrad/functions/photonics/core.py

```diff
@@ -148,7 +148,7 @@ def __init__(
         as_tuple: bool = False,
     ) -> None:
         assert name in ["bragg", "morpho", "chirped"]
-        self.name = name
+        self.name = name + ("as_tuple" if as_tuple else "")
```

Comment:
maybe at least add a coma between the name and "as_tuple"?
as far as I am concerned a better option would be to use `f"{name},{self.parametrization.name}"` although in that case we would need to set a proper name to the parametrization when it's an array, that's not backward compatible (we'd need to update the names in the leaderboard manually)

also, I just figured that in the tuple case, the sigma is not adapted as it is in the array case, that may be a problem.

Reply:
I added a _

I changed the version of mixsimulator, currently master is not ok, this is an issue.

Reply:
Ok even with the current version of mixsimulator this does not work.